miso-web/src/main/webapp/scripts/box_visualization.js

```diff
@@ -56,12 +56,19 @@ var BoxPosition = function(opts) {
   };
 
   self.click = function(event) {
-    if (event.ctrlKey) {
+    if (isMultiSelectClick(event)) {
       self.controlClick();
     } else {
       self.normalClick();
     }
   };
+  
+  isMultiSelectClick = function(event) {
+    function isMac() {
+      return navigator.platform.indexOf('Mac') != -1;
+    }
+    return (isMac() && event.metaKey || !isMac && e.ctrlKey);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to "false"; some subsequent code is never executed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS2583)